tests/test_wideband_dm_data.py

```diff
@@ -121,6 +121,18 @@ def test_dm_jumps(self):
             dm_jump_map[dmj.key_value[0]] = dmj
         for be in all_backends:
             assert all(dm_jump_value[toa_backends == be] == -dm_jump_map[be].quantity)
+        for dmj_param in dm_jump_params:
+            # test derivative function in the dmjump component
+            d_dm_d_dmjump = self.model.d_dm_d_dmjump(self.toas, dmj_param.name)
+            be = dmj_param.key_value
+            assert all(d_dm_d_dmjump[toa_backends == be] == -1.0 * u.Unit(""))
```

Comment:
Should this test that all the others are zero? Also, can you put the derivative test in a separate test method with a descriptive name so we can tell what has failed, if it does?

Reply:
Just changed.